modules/dnn/test/test_caffe_importer.cpp

```diff
@@ -324,7 +324,8 @@ TEST_P(Reproducibility_SqueezeNet_v1_1, Accuracy)
     Mat ref = blobFromNPY(_tf("squeezenet_v1.1_prob.npy"));
     normAssert(ref, out);
 }
-INSTANTIATE_TEST_CASE_P(/**/, Reproducibility_SqueezeNet_v1_1, availableDnnTargets());
+INSTANTIATE_TEST_CASE_P(/**/, Reproducibility_SqueezeNet_v1_1,
+    testing::ValuesIn(getAvailableTargets(DNN_BACKEND_OPENCV)));
```

Comment:
It enables FP16 tests so we can use common thresholds:

```cpp
double l1 = (target == DNN_TARGET_OPENCL_FP16 || target == DNN_TARGET_MYRIAD) ? 4e-3 : 1e-5;
double lInf = (target == DNN_TARGET_OPENCL_FP16 || target == DNN_TARGET_MYRIAD) ? 2e-2 : 1e-4;
```
It should fix the following tests:
```
Reproducibility_SqueezeNet_v1_1.Accuracy/1, where GetParam() = OCL_FP16
Reproducibility_GoogLeNet.Batching/1, where GetParam() = OCL_FP16
```
However the following ones doesn't pass:
```
Reproducibility_GoogLeNet.IntermediateBlobs/1, where GetParam() = OCL_FP16
Reproducibility_GoogLeNet.SeveralCalls/1, where GetParam() = OCL_FP16
```
I think it's OK if we skip them for now for OCL_FP16 target.